qtfred/src/mission/missionsave.cpp

```diff
@@ -3395,7 +3395,8 @@ int CFred_mission_save::save_objects()
 
 			for (SCP_vector<texture_replace>::iterator ii = Fred_texture_replacements.begin();
 				 ii != Fred_texture_replacements.end(); ++ii) {
-				if (!stricmp(shipp->ship_name, ii->ship_name)) {
+				// Only look at this entry if it's not from the table. Table entries will just be read by FSO.
+				if (!stricmp(shipp->ship_name, ii->ship_name) && (!ii->from_table)) {
```

Comment:
Looks good, except here you use `(!ii->from_table)` while everywhere else you use `!(ii->from_table)`.  Do you want to standardize on one format?

Reply:
I knew I had done it differently at some point....